how-to-docs/DNS-how-to.md

```diff
@@ -0,0 +1,11 @@
+## How to get a DNS record for my service
+
+Visit the [acp-dns](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-dns) Gitlab repo
+- To add a record DNS record, create a new aws_route53_record resource in either [records.tf](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-dns/blob/master/terraform/records.tf) or [zones.tf](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-dns/blob/master/terraform/zones.tf)
+
+An aws_route53_record resource template can be found [here](https://www.terraform.io/docs/providers/aws/r/route53_record.html)
+- Clone and branch the acp-dns repo and create your DNS resource
+
+- If delegating a subdomain, please keep it alphabetcal
+
+-  Submit a PR of your branch and it will be created after your PR has been accepted
```

Comment:
Might not be worth repeating stuff from the `acp-dns` readme here.

Personally, I'd be happy with:

> Visit the [acp-dns](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-dns) Gitlab repo and follow the instructions in the README.